CWE/history/provinces/india/2655 - Tura.txt

```diff
@@ -1,5 +1,6 @@
 owner = RAJ
 controller = RAJ
+add_core = MRH
```

Comment:
Would renaming this file cause a crash?

Reply:
If you got a crash, its because there's also a file in the 'soviet' folder. I deleted it to move it, but it didn't push it. I have no idea why - given that its pushed other deletions.